src/extra/fromEvent.ts

```diff
@@ -54,15 +75,39 @@ export class DOMEventProducer implements InternalProducer<Event> {
  * > 'Button clicked!'
  * ```
  *
- * @param {EventTarget} node The element we want to listen to.
- * @param {string} eventType The type of events we want to listen to.
- * @param {boolean} useCapture An optional boolean that indicates that events of
+ * ```js
+ * import fromEvent from 'xstream/extra/fromEvent'
+ * import {EventEmitter} from 'events';
+ *
+ * const MyEmitter = new EventEmitter();
+ * const stream = fromEvent( MyEmitter, 'foo' );
+ *
+ * stream.addListener({
+ *   next: i => console.log(i),
+ *   error: err => console.error(err),
+ *   complete: () => console.log('completed')
+ * });
+ *
+ * MyEmitter.emit( 'foo', 'bar' );
+ * ```
+ *
+ * ```text
+ * > 'bar'
+ * ```
+ *
+ * @param {EventTarget|EventEmitter} element The element upon which to listen.
+ * @param {string} eventName The name of the event for which to listen.
+ * @param {boolean?} useCapture An optional boolean that indicates that events of
  * this type will be dispatched to the registered listener before being
  * dispatched to any EventTarget beneath it in the DOM tree. Defaults to false.
  * @return {Stream}
  */
-export default function fromEvent(node: EventTarget,
-                                  eventType: string,
-                                  useCapture: boolean = false): Stream<Event> {
-  return new Stream<Event>(new DOMEventProducer(node, eventType, useCapture));
+export default function fromEvent( element: EventTarget | EventEmitter,
+                                   eventName: string,
+                                   useCapture: boolean = false): Stream<Event|any> {
+  if ( element instanceof EventEmitter ) {
```

Comment:
Got it. Rx tests for `addListener` method. If that seems reasonable, I'll go that route.